File: app/src/main/java/org/worshipsongs/domain/ServiceSong.java
package org.worshipsongs.domain;

/**
 * @author Vignesh Palanisamy
 * @version 3.x
 */

public class ServiceSong
{
    private String title;
    private Song song;

    public ServiceSong(String title, Song song)
    {
        this.title = title;
        this.song = song;
    }

    public String getTitle()
    {
        return title;
    }

    public void setTitle(String title)
    {
        this.title = title;
    }

    public Song getSong()
    {
        return song;
    }

    public void setSong(Song song)
    {
        this.song = song;
    }
}

File: app/src/main/java/org/worshipsongs/domain/ServiceSong.kt
package org.worshipsongs.domain

/**
 * @author Vignesh Palanisamy
 * @version 3.x
 */

class ServiceSong(var title: String?, var song: Song?)
{

    override fun toString(): String
    {
        return "ServiceSong{" + "title='" + title + '\''.toString() + ", song=" + song + '}'.toString()
    }
}
